frontend/apps/app/features/migrations/components/RadarChart/RadarChart.tsx

```diff
@@ -3,37 +3,39 @@
 import { useMemo } from 'react'
 import styles from './RadarChart.module.css'
 
-export enum CategoryEnum {
-  MIGRATION_SAFETY = 'MIGRATION_SAFETY',
-  DATA_INTEGRITY = 'DATA_INTEGRITY',
-  PERFORMANCE_IMPACT = 'PERFORMANCE_IMPACT',
-  PROJECT_RULES_CONSISTENCY = 'PROJECT_RULES_CONSISTENCY',
-  SECURITY_OR_SCALABILITY = 'SECURITY_OR_SCALABILITY',
-}
+export const CategoryEnum = {
+  MIGRATION_SAFETY: 'MIGRATION_SAFETY',
+  DATA_INTEGRITY: 'DATA_INTEGRITY',
+  PERFORMANCE_IMPACT: 'PERFORMANCE_IMPACT',
+  PROJECT_RULES_CONSISTENCY: 'PROJECT_RULES_CONSISTENCY',
+  SECURITY_OR_SCALABILITY: 'SECURITY_OR_SCALABILITY',
+} as const
+
+export type CategoryEnumType = keyof typeof CategoryEnum
 
 export type ReviewScore = {
   id: number
   overallReviewId: number
   overallScore: number
-  category: CategoryEnum
+  category: CategoryEnumType
 }
 
 type RadarChartProps = {
   scores: ReviewScore[]
 }
 
-const CATEGORIES = Object.values(CategoryEnum)
-const DEFAULT_SCORE = 10
+const CATEGORIES = Object.keys(CategoryEnum) as CategoryEnumType[]
+const DEFAULT_SCORE = 0
```

Comment:
please do not change it.
```suggestion
const DEFAULT_SCORE = 10
```